src/panels/config/integrations/integration-panels/mqtt/mqtt-config-panel.ts

```diff
@@ -95,13 +113,22 @@ class HaPanelDevMqtt extends LitElement {
     }
   }
 
+  private _handleQos(ev: CustomEvent) {
+    const newValue = (ev.target! as any).value;
+    if (newValue >= 0 && newValue !== this.qos && localStorage && this.inited) {
+      this.qos = newValue;
+      localStorage["panel-dev-mqtt-qos"] = this.qos;
```

Comment:
We should use the `@LocalStorage` decorator :-) (doesn't have to be in this PR)

Reply:
Ah, good follow up PR then.